src/main/java/htsjdk/variant/utils/SAMSequenceDictionaryExtractor.java

```diff
@@ -124,6 +160,17 @@ static TYPE forFile(final File dictionaryExtractable) {
             throw new SAMException("Cannot figure out type of file " + dictionaryExtractable.getAbsolutePath() + " from extension. Current implementation understands the following types: " + Arrays.toString(TYPE.values()));
         }
 
+        static TYPE forPath(final Path dictionaryExtractable) {
```

Comment:
The enum should really be called `Type` as uppercase is typically reserved for static final objects.

Reply:
sorry...that ship has sailed....I can file it under "things to do in htsjdk3"...